src/utils/generateColors.ts

```diff
@@ -0,0 +1,27 @@
+import randomSeed from 'random-seed';
+
+export type SeededColorCombination = {
+    bg: string;
+    fg: string;
+}
+
+export function generateColors(seed: string): SeededColorCombination {
+    const rand = randomSeed.create(seed.toString());
```

Comment:
If the `seed` function argument is already a string, doing `toString()` should not be necessary. If doing `toString()` is necessary, the type is not correct.